tests/Functional/EnglishHumanizationTest.php

```diff
@@ -33,6 +33,10 @@ public function humanizationProvider(): Generator {
 
 		yield 'Month only' => [ 'XXXX-12-XX', 'December' ];
 		yield 'Day only' => [ 'XXXX-XX-12', '12th' ];
+		
+		// https://github.com/ProfessionalWiki/EDTF/issues/80
+		yield 'Some year' => [ 'XXXX', 'some year' ];
```

Comment:
Maybe add
yield 'Some year' => [ 'X', 'some year' ];
yield 'Some year' => [ 'XXX', 'some year' ];

Curious as to what "-X" would yield.

Reply:
> And check what -5XXXX results into, unless already tested

50000 BC

this is nice, but still doesn't account for the unspecified digits

Reply:
it should perhaps be:

50000 BC (millennium, century and year in decem millennium unspecified)

Reply:
That does not scale. Because what is -5XXXXXXX ? How about 50000 BC (year unspecified)

Reply:
That means implementing scales (https://en.wikipedia.org/wiki/Long_and_short_scales#Short_scale) e.g. the short scale to billion? As (according to Google) the universe is 13 billion years old and will likely exists for 22 billion years and this is about dates. :)

Reply:
I was checking further the specification here https://www.loc.gov/standards/datetime/
(Unspecified Digit) and the parser,
but I'm not sure that the parser is checking whether
the unspecified digit is placed incrementally from the right of the date component.
e.g.
the following should throw an error:

-XXXXXXX4 (4th year of any millennium BC)